Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -422,7 +422,16 @@ export async function getP12SHA1Hash(p12Path: string, p12Pwd: string) {
             }
         }
     })
-    await openssl1.exec();
+
+    try {
+        await openssl1.exec();
+    } catch (err) {
+        if (!p12Pwd) {
+            tl.warning(tl.loc('NoP12PwdWarning'));
```

Comment:
I wanted to make it more obvious when you didn't supply a P12 password, particularly with the new Xcode template which reads the password from a process variable.

Reply:
We discussed this in person. You only see this warning if openssl failed and you didn't provide a password. I think it's easy to forget to visit the process variables tab when your creating a new definition from the Xcode template.